src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -153,10 +157,12 @@ def _update_expiration_warnings(self, loop_start_time):
             self.expiration_warning_message = new_expiration_warning_message
 
             # Actually send the warning e-mail.
-            message_dict = {'subject': self.config['default_subject'],
-                'body': 'The expiration status of one or more keys have changed.'}
-            encrypted_message = self._build_encrypted_message(message_dict, loop_start_time)
-            self.mailsender.sendmail(encrypted_message, self.valid_recipient_emails)
+            for email in self.gpgkeyverifier.valid_recipient_emails:
+                message_dict = {'subject': self.config['default_subject'],
+                    'body': 'The expiration status of one or more keys have changed.',
+                    'recipients': email['email']}
```

Comment:
Why do we add the e-mail to the message_dict?

Reply:
Probably don't need to since this is where the expiration messages are sent, not the messages directed by the caller